crates/compiler/mono/src/drop_specialization.rs

```diff
@@ -1059,6 +1250,10 @@ impl<'a> DropSpecializationEnvironment<'a> {
             children.iter().for_each(|child| res.push(*child));
         }
 
+        if let Some(children) = self.list_children.get(parent) {
+            children.iter().for_each(|(child, _)| res.push(*child));
```

Comment:
I think this could instead be a `res.extend(children.iter().map(|(child, _)| child)`, which I'd consider more idiomatic